.github/workflows/publish.yml

```diff
@@ -24,6 +24,13 @@ jobs:
         uses: manovotny/github-releases-for-automated-package-publishing-action@v2.0.1
 
       # Setup .npmrc file to publish to npm
+      - name: Setup Anchor & Build
```

Comment:
Maybe move the comment above down to above npm?

Reply:
yup, sorry. I messed that up. have fixed it.